dcap-ql/src/lib.rs

```diff
@@ -20,6 +22,8 @@ extern crate libc;
 #[macro_use]
 extern crate num_derive;
 extern crate num_traits;
+#[cfg(feature = "serde_support")]
+extern crate serde;
```

Comment:
`extern crate` is no longer needed in most cases in Rust 2018, but this is an issue separate from your PR :). In the future it might be worth another PR to change the `edition` so we can get rid of the instances `extern crate` and `macro_use`.